pkg/plugin/karmada/plugin.go

```diff
@@ -94,6 +100,11 @@ func (p *KarmadaPlugin) runInstall() error {
 	installArgs := []string{
 		"init",
 	}
+
+	for _, flag := range p.args.SetFlags {
+		installArgs = append(installArgs, fmt.Sprintf("--%s", flag))
```

Comment:
what would happen if we pass some invalid flags, e.g. `--set cert-external-errdns=xxxxx`

Reply:
`karmada init ` returns `Error: unknown flag: --cert-external-errdns`

Reply:
```bash
root@kurator-test-01:~# ./kurator install karmada --set cert-external-errdns=xxxx  --kubeconfig .kube/config 
Error: unknown flag: --cert-external-errdns
ERRO[2022-06-29 15:38:24] failed to install karmada, exit status 1     
ERRO[2022-06-29 15:38:24] karmada execute error: exit status 1         
Error: karmada execute error: exit status 1
execute kurator command failed:  karmada execute error: exit status 1
```

Reply:
I'm thinking about pass args without `set`
```console
./kurator install karmada --cert-external-errdns=xxxx
vs
./kurator install karmada --set cert-external-errdns=xxxx
```

cc @hzxuzhonghu WDYT?

Reply:
>./kurator install karmada --set cert-external-errdns=xxxx

It looks simpler

Reply:
One question: is it possible to get the args that are not unknown? IIUC, in order to remove `--set `, we need to pass all unknown args to karmada init, right?

Reply:
It seems to me that karmada init acts as a flag receiver and only it knows which flags are unknown. This has nothing to do with `--set` or not. 

The problem is that kurator has no good way to verify which karmada init flags are unknown or invalid.